code/include/observations.h

```diff
@@ -75,56 +77,85 @@ namespace picopter {
         CAMERA_FLOW,    //optical flow will resolve a range and blob (probably include two images and IMU velocity)
```

Comment:
If you want your comments to show up in documentation, you should use Doxygen style comments, e.g.

~~~~~
/** Optical flow will resolve a range and blob (probably include two images and IMU velocity) **/
~~~~~